CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## v.1.7.1
+## v1.7.2
+
+- Bugfix for Plugwise-beta Issue [#833](https://github.com/plugwise/plugwise-beta/issues/833)
```

Comment:
I'd suggest we mention it's not only beta - but rather solving an issue with the stretch? + leave the hash out :)